receiver/mongodbatlasreceiver/README.md

```diff
@@ -75,6 +75,20 @@ MongoDB Atlas [Documentation](https://www.mongodb.com/docs/atlas/reference/api/l
     - This will limit how many pages of events the receiver will request from the MongoDB Atlas API for each project.
   - `types` (defaults to all types of events)
     - This is a list of [event types](https://www.mongodb.com/docs/atlas/reference/api/events-orgs-get-all/#event-type-values) that the receiver will request from the API. If specified, the receiver will collect only the indicated types of events.
+- `access_logs`
```

Comment:
It's a bit confusing to me that access logs are a separate section from other logs. Is this really necessary? It seems to introduce a lot of unnecessary duplication.

We already include audit logs as an optional type of logs. Wouldn't it make sense to include access logs there too?

Reply:
We lose page size controls if we swap access logs to just a flag in project & embed in logs. We could then add access_log_page_size and access_log_max_pages, does that still feel like the right direction to you?

Reply:
There's also the possibility that a user would want access logs, but not the rest of the logs. I don't see a way to configure that in the combined config path.